### ISSUE TITLE
Phantom Thrust  be used to pull a party member

### ISSUE THREAD

**Issue by julia40124009** [rathena/rathena#340]
http://irowiki.org/wiki/Phantom_Thrust

"This skill can be used to pull a party member without inflicting damage"

seem no work with party member

version https://github.com/rathena/rathena/commit/bae469e6c6e45a7a08db9c27f31e249eaa52207d
client 2013-12-23

**Comment by cydh** [comment]
interesting

**Comment by cydh** [comment]
can u just change on skill_db.txt and test it, works or not. xD
````
-2020,5:6:7:8:9,6,1,-1,0,0,5,1,no,0,0,0,weapon,0,0x0,	RK_PHANTOMTHRUST,Phantom Thrust
+2020,5:6:7:8:9,6,17,-1,0,0,5,1,no,0,0,0,weapon,0,0x0,	RK_PHANTOMTHRUST,Phantom Thrust
````

**Comment by julia40124009** [comment]
skill can't use

**Comment by cydh** [comment]
should be fixed in 46f010cb28d3785d0c89e394b382a7e39f01a188
